code/modules/mob/living/carbon/xenomorph/abilities/warrior/warrior_abilities.dm

```diff
@@ -1,17 +1,3 @@
-// toggle agility
-/datum/action/xeno_action/onclick/toggle_agility
```

Comment:
if you are going to remove agility you should remove it completely

Reply:
At the time I did this the plan was to remove `var/agility` from xenomorph class in refactor of xenomorph.dm as I was unsure if it was modified elsewhere as it is also referenced in prae code. But having looked deeper it looks like I can just remove all checks for `agility` in Warrior code.